File: src/main/java/ru/pashkovske/buratino/tinkoff/service/common/utils/TimeUtils.java
package ru.pashkovske.buratino.tinkoff.service.common.utils;

import com.google.protobuf.Timestamp;

import java.time.Instant;

public class TimeUtils {
    public static Instant map(Timestamp ts) {
        return Instant.ofEpochSecond(ts.getSeconds(), ts.getNanos());
    }
}

File: src/main/kotlin/ru/pashkovske/buratino/common/utils/TimeUtils.kt
package ru.pashkovske.buratino.common.utils

import com.google.protobuf.Timestamp
import java.time.Instant

object TimeUtils {
    fun tsToInstant(ts: Timestamp): Instant {
        return Instant.ofEpochSecond(ts.seconds, ts.nanos.toLong())
    }
}